README.md

```diff
@@ -1 +1,172 @@
-# rust-vmm-ci
\ No newline at end of file
+# rust-vmm-ci
+
+The `rust-vmm-ci` contains integration tests and [Buildkite]() pipeline
+definitions that are used for running the CI for all rust-vmm crates.
+
+Having a centralized place for the tests is one of the enabler for keeping the
+same quality standard for all crates in rust-vmm.
+
+## Getting Started with rust-vmm-ci
+
+To run the integration tests defined in the pipeline as part of the CI:
+
+1. Add rust-vmm-ci as a git submodule to your repository
+
+```bash
+git submodule add https://github/rust-vmm/rust-vmm-ci
+git add rust-vmm-ci/
+git commit -s -m "Added rust-vmm-ci as submodule"
+```
+
```

Comment:
You may want to briefly what that does and why we do this.

Reply:
Do you mean the submodule?

Reply:
I mean to explain what does it actually do and why we're doing it. A readable explanation after the set of commands.
But that's a nit.